scene/gui/graph_node.cpp

```diff
@@ -665,18 +668,30 @@ void GraphNode::_port_pos_update() {
 
 		Size2i size = child->get_rect().size;
 
-		if (slot_table.has(slot_index)) {
+		if (slot_table.has(slot_index)) {		
 			if (slot_table[slot_index].enable_left) {
 				PortCache port_cache;
-				port_cache.pos = Point2i(edgeofs, vertical_ofs + size.height / 2);
+				port_cache.pos = Point2i(edgeofs, vertical_ofs + size.height / 2);	
+				// Apply rotation
```

Comment:
```suggestion
				// Apply rotation.
```
Per the [comment style](https://docs.godotengine.org/en/latest/contributing/development/code_style_guidelines.html#comment-style-guide).